spec/models/bus_stop_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'spec_helper'
+
+describe BusStop do
+  describe 'decide_if_completed_by' do
+    let(:user) { create :user }
+    let(:stop) { create :bus_stop, :pending }
+    context 'bus stop completed attribute changed' do
+      context 'bus stop is completed' do
+        it 'assigns user to completed by' do
+          Timecop.freeze Date.today do
```

Comment:
Curious why we need to freeze to today.

Reply:
As opposed to what? Freezing to tomorrow? Or are you saying you want to know why I'm freezing.

Reply:
Yes. why freeze at all, I'm not catching

Reply:
Oh, in order to see that the attribute has changed.

Reply:
Still confused. What happens if you don't freeze, I guess is what I'm asking.

Reply:
No difference 🐑